osc-server/src/main/java/org/osc/core/server/Server.java

```diff
@@ -118,7 +118,7 @@
 
     // static to avoid many trivial references
     private static Integer apiPort = DEFAULT_API_PORT;
-    private static volatile boolean inMaintenance = false;
+    private volatile boolean inMaintenance = false;
```

Comment:
Is this still being globally stored? If multiple instances of the Server  gets created they each will have their own inMaintenance value right?